docker/transport/basehttpadapter.py

```diff
@@ -0,0 +1,6 @@
+import requests.adapters
+
+
```

Comment:
nit: Extra `\n`

Reply:
Linter says that before a class it needs 2 empty lines